src/components/Input/InputCard.tsx

```diff
@@ -71,12 +85,18 @@ export default function InputCard({
     }
   };
 
+  const isAddDisabled = () => {
+    if (type === "card" && (title === "" || user === "")) return true;
+    else if (title === "") return true;
```

Comment:
it would be handled in the above condition!

Reply:
done